contrib/pgtap/Trunk.toml

```
@@ -0,0 +1,20 @@
+[extension]
+name = "pgtap"
+version = "1.2.1"
```

This should be `1.2.0` for the release we're using: https://github.com/theory/pgtap/blob/6ca7aec6d7e9ddca05fcecba3ac9132a446e2511/pgtap.control#L3C19-L3C19
```suggestion
version = "1.2.0"
```